packages/manager/modules/pci/src/projects/project/storages/databases/fork/fork.html

```diff
@@ -0,0 +1,195 @@
+<oui-header data-heading="{{:: 'pci_database_fork_title' | translate }}">
+    <oui-guide-menu data-text="{{:: 'pci_project_guides_header' | translate }}">
```

Comment:
If the view is similar to the order database page (the existing page), we will have to try to reuse it as a component

Reply:
The views forks and add databases are quite different, and will diverge even more with future lot, that is why they are separated files.